ietf/status/views.py

```diff
@@ -0,0 +1,50 @@
+# Copyright The IETF Trust 2016-2020, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+import json
+from django.http import HttpResponse, HttpResponseRedirect, HttpResponseNotFound
+from ietf.utils import markdown
+from django.core.exceptions import FieldError
+from django.shortcuts import render, get_object_or_404
+from ietf.status.models import Status
+
+import debug                            # pyflakes:ignore
+
+def get_context_data():
+    status = Status.objects.order_by("-date").first()
+    if status is None or status.active == False:
+        return { "hasMessage": False }
+
+    if status.slug is None:
+        raise FieldError("No slug generated. This shouldn't happen.")
+
+    context = {
+        "hasMessage": True,
+        "id": status.id,
+        "slug": status.slug,
+        "title": status.title,
+        "body": status.body,
+        "url": f"/status/{status.slug}",
+        "date": status.date.isoformat(),
+        "by": status.by.name,
```

Comment:
`by` seems to be inconsistently used? The admin could default it to the logged in user, but maybe we should remove it from the model?

Reply:
I've removed it from the templates / API response, but kept it in the model for auditing purposes. Happy to remove it if you'd prefer though.